src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/DebugPropertyPage/RemoveStartArgumentsActionHandler.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.Xml;
+using Microsoft.VisualStudio.ProjectSystem.VS.PropertyPages.Designer;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties;
+
+[Export(typeof(ILinkActionHandler))]
+[ExportMetadata("CommandName", "RemoveStartArguments")]
+internal class RemoveStartArgumentsActionHandler : ILinkActionHandler
```

Comment:
It would be good if this action handler could move or append the value it is removing to the argument list in the `launchSettings.json` too.

Reply:
I'll wait for feedback from users on this proposal.
Users can still manually copy and paste it at the front or at the back.

Reply:
Yes, but once they click the hyperlink the value will disappear from screen and there is no undo feature. I would find that very frustrating.

Would it be hard to fix?

Reply:
done